schema/pet-insurance.json

```diff
@@ -55,86 +61,129 @@
         },
         "finalPrice": {
             "description": "Emitted immediately before placing order, when final price is available.<br/>Automation will request <code>finalPriceConsent</code> input which should match this object.",
             "typeRef": "#/domains/Generic/types/PriceConsent"
         },
         "availableMaritalStatusOptions": {
+            "description": "Owner's marital status available options",
             "typeRef": "#/domains/PetInsurance/types/MaritalStatusOptions"
         },
         "availableVoluntaryExcesses": {
+            "description": "Available voluntary excesses (in % increments)",
             "typeRef": "#/domains/PetInsurance/types/VoluntaryExcesses"
         },
         "availableAddresses": {
+            "description": "Available addresses after postcode input",
             "typeRef": "#/domains/PetInsurance/types/Addresses"
         },
         "availableCovers": {
+            "description": "Available protection covers.",
             "typeRef": "#/domains/PetInsurance/types/Covers"
         },
         "availableCoverTypes": {
+            "description": "Available types of coverage",
             "typeRef": "#/domains/PetInsurance/types/CoverTypes"
         },
-        "availableVetFeesLimit": {
-            "typeRef": "#/domains/PetInsurance/types/VetFeesLimit"
+        "availableVetFees": {
+            "description": "Availebel vet fee amounts covered",
+            "typeRef": "#/domains/PetInsurance/types/VetFees"
         },
         "availableCoverOptions": {
+            "description": "Available optional extra packages",
             "typeRef": "#/domains/PetInsurance/types/CoverOptions"
         },
         "availableVetPaymentTerms": {
-            "typeRef": "#/domains/PetInsurance/types/VetPaymentTerms"
+            "description": "Available types of vet bill coverage",
+            "typeRef": "#/domains/PetInsurance/types/PaymentTerms"
         },
         "availablePaymentTerms": {
+            "description": "Available methods of payment (card, directdebit etc)",
             "typeRef": "#/domains/PetInsurance/types/PaymentTerms"
         },
+        "insuranceProductInformationDocument": {
+            "typeRef": "#/domains/PetInsurance/types/NullableStructuredTextItem",
+            "description": "Emitted when the document inforamtion is presented."
+        },
+        "essentialInformation": {
+            "typeRef": "#/domains/PetInsurance/types/NullableStructuredTextItem",
+            "description": "Emitted when the esential information document is presented."
+        },
+        "policyWording": {
+            "typeRef": "#/domains/PetInsurance/types/NullableStructuredTextItem",
+            "description": "Emitted when the policy wording is presented."
+        },
         "purchaseConfirmation": {
             "typeRef": "#/domains/PetInsurance/types/PurchaseConfirmation"
+        },
+        "availableDirectDebitPaymentDays": {

```

Comment:
The input is `selectedPaymentDay`, shouldn't this output be called `availablePaymentDays`?

Reply:
Make sense.